build.yaml

```diff
@@ -1021,6 +1029,7 @@ libs:
   - grpc++_codegen_base_src
   - grpc++_codegen_proto
   - grpc++_config_proto
+  - thrift_util
```

Comment:
So far, there are no tests for thrift. This dependency on thrift will not effect any C/C++ builds as long as these files thrift_utils.h , thrift_serializer.h , thrift_serializer_inl.h are present in include/grpc++/impl/codegen.  There are cpp examples which can be used to test the integration. install thrift from third_party/thrift and run example in tutorial/cpp.